src/sdk/pynni/nni/trial.py

```diff
@@ -138,5 +137,5 @@ def report_final_result(metric):
         'type': 'FINAL',
         'sequence': 0,
         'value': metric
-    })
+    }, allow_nan=True)
```

Comment:
I think we should have `nni.utils.to_json = functools.partial(json_tricks.dumps, allow_nan=True)`.

Reply:
Good suggestion, thanks!

Reply:
Fixed.